src/FormBuilder/resources/js/form-builder/src/components/modal/index.tsx

```diff
@@ -1,10 +1,10 @@
-import {Button, Icon, Modal as GutenbergModal} from '@wordpress/components';
+import {Button, Icon} from '@wordpress/components';
+import GutenbergModal from '@wordpress/components/modal';
```

Comment:
Can we try this instead?

```suggestion
import {Button, Icon, Modal as GutenbergModal} from '@wordpress/components';

```

Reply:
I have tried that before getting to this solution.

![CleanShot 2023-10-02 at 13 28 58](https://github.com/impress-org/givewp/assets/3921017/75ef0086-6ec1-4634-afd4-fc9fca0f1613)

Reply:
Should we import the type directly?

```suggestion
import {Button, Icon} from '@wordpress/components';
import GutenbergModalProps from '@wordpress/components/modal/types';
```

Reply:
Hm... It looks like we need to import from `@wordpress/components/modal` because of how Gutenberg defines the namespace, which is lost when imported/exported through `@wordpress/components`.

Reply:
The tricky part is that wordpress scripts has no concept of subfolders so  the import needs to be from `@wordpress/components` or else it won't work properly.  I'm thinking we just go back to importing the modal from the main ``@wordpress/components`` package and pull in the types manually from gutenberg.